docker/README.md

```diff
@@ -83,17 +84,6 @@ $ docker-compose run btcctl generate 400
 $ docker-compose run btcctl getblockchaininfo | grep -A 1 segwit
 ```
 
-Now we have `btcd` running and some amount of bitcoins mined to the `Alice`
```

Comment:
We talked offline, and Andrew pointed out that since we now auto-reconnect to `btcd` within `btcwallet`, this is no longer needed.